src/Bridges/bridge.jl

```diff
@@ -0,0 +1,17 @@
+abstract type AbstractBridge end
+
+"""
+    added_constraint_types(BT::Type{<:Variable.AbstractBridge})::Bool
+
+Return a list of the types of constrained variables that bridges of concrete
+type `BT` add. This is used by the [`LazyBridgeOptimizer`](@ref).
+"""
+function added_constrained_variable_types end
```

Comment:
This should be a generic fallback
```Julia
function added_constrained_variable_Types(::Type{<:Variable.AbstractBridge})
    return Tuple{DataType}[]
end
```

Reply:
It may be dangerous because if someone forget to implement the method, the only effect is that Bellman-Ford might work incorrectly.
That may produce nasty bugs